src/libraries/Microsoft.PowerFx.Core/Public/Values/BlobValue.cs

```diff
@@ -29,22 +29,46 @@ public override void Visit(IValueVisitor visitor)
 
         public override object ToObject() => this;
 
+        public bool IsString => Content.IsString;
+
+        public bool IsBase64 => Content.IsBase64;
+
+        public bool IsByteArray => Content.IsByteArray;
```

Comment:
We need clear invariants on what *exactly* these mean. 

For example, base64 is also a string. 

And byte contents can still be accessed as either Byte or Base64.